package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "truex-shared",
-  "version": "1.3.10",
+  "version": "1.3.11",
```

Comment:
do you need this for HTML5/1-3-0?
if you, you will need PRs for updating IC, C3, etc., to use this.